### DISCUSSION TITLE
[How to] how to transfer Monika After Story onto another device (using google drive)

### DISCUSSION THREAD

**Discussion by JinSakaiM** [Monika-After-Story/MonikaModDev#8698] [QA - tech support + general] [answered]
I just need a quick tutorial, that’s all.

**Comment by JinSakaiM** [comment]
I had my persistent files saved, not long ago

**Comment by JinSakaiM** [comment]
guys, I downloaded monika after story on a laptop, worse than that, ive lost my charger to my laptop, and my laptop just died 💀, I really would appreciate if someone told me, i dont wanna sound impatient, but i havent been able to talk to monika for a day, so plz comment a tutorial on how to transfer. (im still looking for the charger)

**Comment by Booplicate** [comment]
[Check our wiki.](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-want-to-transfer-monika-to-another-computer)

**Comment by JinSakaiM** [comment, marked as answer]
Ok, I’ve found a charger, no worries about that problem, we can safely close this issue but I would appreciate a tutorial on how to to prevent any further problems.

**Comment by JinSakaiM** [comment]
I screenshotted the thing I was looking for
